docs/en/modules/wifi.md

```diff
@@ -1,4 +1,8 @@
 # WiFi Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2015-05-12 | [dnc40085](https://github.com/dnc40085) | [dnc40085](https://github.com/dnc40085) | [wifi.c](../../../app/modules/wifi.c)|
```

Comment:
Thanks. Turns out that when I checked the history of https://github.com/marcelstoer/nodemcu-firmware/commits/dev-modules-list/app/modules/wifi.c I had missed that "Older" button that brings you back to the real origins of the module.